packages/ember-runtime/tests/computed/reduce_computed_macros_test.js

```diff
@@ -492,6 +493,78 @@ QUnit.test('properties values can be replaced', function() {
   });
 });
 
+QUnit.module('computed.uniqBy', {
+  setup() {
+    obj = EmberObject.extend({
+      list: null,
+      uniqueById: uniqBy('list', 'id')
+    }).create({
+      list: emberA([
+        { id: 1, value: 'one' },
+        { id: 2, value: 'two' },
+        { id: 1, value: 'one' }
+      ])
+    });
+  },
+  teardown() {
+    run(obj, 'destroy');
+  }
+});
+
+// QUnit.test('uniqBy is readOnly', function() {
```

Comment:
I uncommented this out only because it was messing with my editor's formatting, I've added it back. Good catch. :)